source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -65,6 +72,7 @@ RoleBasedAccessControlFilterConfig::RoleBasedAccessControlFilterConfig(
                                                   proto_config.shadow_rules_stat_prefix(), scope)),
       shadow_rules_stat_prefix_(proto_config.shadow_rules_stat_prefix()),
       per_rule_stats_(proto_config.track_per_rule_stats()),
+      deny_status_(getDenyStatus(proto_config.deny_status())),
```

Comment:
Maybe explicitly do a static cast to uint16_t for the argument? I think the overflow never really happens but the sanitizers will probably complain if you implicitly convert values.